client/components/boards/boardsList.styl

```diff
@@ -128,3 +128,26 @@ $spaceBetweenTiles = 16px
         font-size: 25px
         color: white
 
+@media screen and (max-width: 800px)
+  .board-list
+    height: 100%
+    overflow: scroll
+
+    li
+      width: 33.3%
+
+    .board-list-item
+      overflow: hidden
+
+    .board-list-item-sub-name
+      position: relative
+      top: -100px
+      left: -100px
+
+@media screen and (max-width: 360px)
+  .board-list
+    height: 100%
+    overflow: scroll
```

Comment:
Is there a need to repeat these 3 lines since there are already implied above by `max-width: 800px`